HabboHotel/Groups/GroupManager.cs

```diff
@@ -177,23 +177,19 @@ public List<Group> GetGroupsForUser(int userId)
             List<Group> groups = new List<Group>();
             using (IQueryAdapter dbClient = PlusEnvironment.GetDatabaseManager().GetQueryReactor())
             {
-                dbClient.SetQuery("SELECT g.id FROM `group_memberships` AS m RIGHT JOIN `groups` AS g ON m.group_id = g.id WHERE m.user_id = @user");
+                dbClient.SetQuery("SELECT g.id FROM `group_memberships` AS m RIGHT JOIN `groups` AS g ON m.group_id = g.id WHERE m.user_id = @user ORDER BY g.id DESC");

```

Comment:
This doesn't really make a difference, all it does it sorts the table, when selecting.